server/src/main/java/org/elasticsearch/index/fielddata/plain/AbstractLatLonPointDVIndexFieldData.java

```diff
@@ -41,10 +41,29 @@
 import org.elasticsearch.search.sort.BucketedSort;
 import org.elasticsearch.search.sort.SortOrder;
 
-public abstract class AbstractLatLonPointDVIndexFieldData extends DocValuesIndexFieldData
-    implements IndexGeoPointFieldData {
+public abstract class AbstractLatLonPointDVIndexFieldData implements IndexGeoPointFieldData {
```

Comment:
What would you think about removing 'DV' from all of these class names? Now that they don't extend `DocValuesIndexFieldData`, I think it's a bit confusing/ unclear what 'DV' refers to.

Reply:
Yeah, that seems reasonable, will do.